src/api/materials.py

```
@@ -0,0 +1,541 @@
+    Parse a multipart/form-data request using the CF Workers FormData API.
+
+    Returns ``(fields_dict, error_response | None)`` where *fields_dict*
+    maps field names to either a string value (text fields) or a dict
+    ``{"filename": str, "bytes": bytes}`` (file fields).
+
+    Falls back to a plain JSON body when the Content-Type is
+    ``application/json`` so that unit tests can exercise the handler
+    without constructing real multipart payloads.
+    """
+    w = _worker()
+    content_type = (req.headers.get("Content-Type") or
```

Flake8 (7.3.0)

[warning] line break after binary operator

(W504)